app.py

```diff
@@ -552,6 +552,9 @@ def error(self, code, message, **kwargs):
 
             self.filedata = body
 
+            if self.request_id[:5] == "00000":
```

Comment:
i don't see anything wrong with this code either.  

The same suggestions apply.  I would try to wrap the prefix comparison logic in a method.  Basically I would try to make it more clear what the intentions of this code is.